sleap/gui/commands.py

```diff
@@ -3901,3 +3905,20 @@ def copy_to_clipboard(text: str):
     clipboard = QtWidgets.QApplication.clipboard()
     clipboard.clear(mode=clipboard.Clipboard)
     clipboard.setText(text, mode=clipboard.Clipboard)
+
+class UnlinkVideo(EditCommand):
+    @staticmethod
```

Comment:
We are missing the callback `topics` here.
```suggestion
class UnlinkVideo(EditCommand):
    topics = [UpdateTopic.sessions]
    
    @staticmethod
```